stage1/init/kvm/hypervisor/none_driver.go

```diff
@@ -0,0 +1,27 @@
+// Copyright 2016 The rkt Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// +build hv_none
```

Comment:
@alban:
*You are building rkt without kvm support, and then you are passing kvm stage1 aci file to run container.
https://github.com/coreos/rkt/pull/2684/files#diff-67e997bcfdac55191033d57a16d1408aR418
https://github.com/coreos/rkt/pull/2684/files#diff-853b610e48968edda7ac95f276f624f1R15
 It's also default code if you do not want to have kvm support.